scripts/benchmark/utils/format.py

```diff
@@ -0,0 +1,14 @@
+from evm.rlp.blocks import BaseBlock
+from evm.utils.hexadecimal import (
+    encode_hex,
+)
+
+
+def format_block(block: BaseBlock) -> str:
+    return (
+        "\n\n"
+        "------------------------Block------------------------------------\n"
+        "Number #{b.number}\n"
+        "Hash {hash}\n"
```

Comment:
nitpick, but these could probably be put on the same line if you put the number first and print it with a fixed-width of about 10-12 characters depending on how future proof you want to be.